crates/runix/src/flake_ref.rs

```diff
@@ -171,6 +178,49 @@ impl ToFlakeRef {
             "flake" => ToFlakeRef::Indirect(IndirectFlake {
                 id: url.path().to_string(),
             }),
+            git_scheme if git_scheme.starts_with("git+") => {
```

Comment:
I feel like this would be more readable/grepable and also more correct
```suggestion
            "git+http" | "git+https" | "git+ssh" | "git+file" => {
```

Reply:
indeed